_extensions/posit-docs/theme-dark.scss

```diff
@@ -0,0 +1,395 @@
+$font-family-monospace: "Source Code Pro", monospace;
+$font-family-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$font-size-base: 1rem;
+// scss-docs-end font-variables
+
+//font and body color
+
+$body-color: $dark-md-body;
+$popover-bg: $posit-green !default; // check this - citation previews
+
+// scss-docs-start font-sizes - seems that I have to keep these in the style sheet?
+$h1-font-size:44px;
+$h2-font-size:28px;x
+
+h1 {
+  size: $h1-font-size;
+}
+h2 {
+  size: $h2-font-size;
+}
+// scss-docs-end font-sizes
+
+// Headings
+// scss-docs-start headings-variables
+$headings-color: #ffffff;
+// scss-docs-end headings-variables
+
+//  Customize the navbar
+$navbar-bg: lighten($dark-md-background, 5%);
+$navbar-fg: #ffffff; 
+
+// CSS overrides
+
+// Body
+//
+// Settings for the `<body>` element.
+
+//$body-color: $posit-green !default;
+$body-bg: $dark-md-background !default; // dark mode change
+
+
+// Nav and footer
+
+/* Nav bar */
+
+.navbar-title {
+  color: #ffffff; // dark theme change
+}
+
+.nav-link {
+font-family: $font-family-monospace;
+text-transform: uppercase;
+letter-spacing: .03em;
+font-size: 14px !important;
+font-weight: 500 !important;
+color: #fff !important;
+}
+
+.navbar-nav .nav-link.active,
+.navbar-nav .nav-link.show {
+color: $posit-teal !important;
+font-weight: 700 !important;
+}
+
+/* Nav bar  right hamburger menu */
+
+.dropdown-item:hover, .dropdown-item:focus {
+color: var(--bs-dropdown-link-hover-color);
+background-color: #0F1C29 !important;
+}
+
+/* Footer */
+
+.nav-footer {
+  color:  #ffffff !important;
+  border-top: solid $posit-gray 1px;
+}
+
+/* Footer icons */
+.nav-footer a {
+color: #ffffff !important;
+}
+
+
+// Footnotes
+
+.footnote-back {
+  color: $posit-light-teal-2;
+}
+
+
+// Breadcrumbs 
+
+.quarto-title-breadcrumbs .breadcrumb li:last-of-type a {
+  color: $posit-light-gray-1 !important;
+}
+
+// Page navigation
+
+.nav-page .nav-page-text {
+  color: $posit-light-gray-1;
+}
+
+// Code
+
+/* Code block elements */
+
+pre > code.sourceCode > span {
+  //color: #003B4F;
+}
+
+code span.ex {
+    //font-weight: bold;
+    color: #ffffff;
+}
+
+code span.dt {
+  color: #FFABCA;
+    //font-style: inherit;
+}
+
+code span.ot {
+    //color: rgb(60, 222, 222);
+}
+
+code span.st {
+    //color: greenyellow;
+}
+
+/* Inline code */
+p code:not(.sourceCode), li code:not(.sourceCode), td code:not(.sourceCode) {
+  background-color: #6e768166 !important;
+  color: #ffffff;
+}
+
+// Code blocks
+
+/* Code block title*/
+
+.quarto-dark .code-with-filename .code-with-filename-file {
+  background-color: #000000 !important;
+  border: none;
+}
+
+.code-with-filename strong {
+  color: #6E9AC3;
+}
+
+/* Code block background */
+$code-block-bg: lighten($dark-md-background, 10%);
+
+//$monobackgroundcolor: $dark-md-background !default;
```

Comment:
here too re: commented out code?

Reply:
I think that I am ok to remove this as well